include/envoy/server/filter_config.h

```diff
@@ -206,7 +206,8 @@ class FactoryContext : public virtual CommonFactoryContext {
 class FilterChainFactoryContext : public virtual FactoryContext {
 public:
   /**
-   * @return uint64_t The tag assigned with this context at runtime.
+   * @return uint64_t The tag assigned with this context at runtime. It will be used for connection
+   * handler tracking.
```

Comment:
I suggest referencing the relevant listener tag intro comments and terms as well.